src/nuxeo-cloud/what-is-nuxeo-cloud.md

```diff
@@ -0,0 +1,208 @@
+---
+title: What's Nuxeo Cloud
+review:
+    comment: ''
+    date: '19-03-2020'
+    status: ok
+labels:
+    - nuxeo-cloud
+tree_item_index: 100
+toc: true
+---
+
+Nuxeo Cloud is a fully hosted and managed offering providing our customers with setup, support and maintenance for a complete Nuxeo implementation.
+
+This includes support and management of the infrastructure, third-party services, and application, including all deployments and/or changes to production and pre-production environments.
+
+![]({{file name='cloud-architecture.png'}})
+
+## Nuxeo Cloud Environments
+
+The Nuxeo Cloud Operations (NCO) team provides different environments as part of a standard configuration with the latest LTS and Hotfixes applied.
+
+### Production
+
+Primary environment used by customers:
+
+- Full stack deployment of Nuxeo and all required services
+- Complete monitoring and support
+- PCI/DSS and SOC2 Certified (as of 2019)
+
+### Pre-Production
+
+Environment used to validate deployments before going to production. These deployments may include cloud platform changes, hotfixes and security patches:
+- Reduced stack deployment of Nuxeo to mirror production
+- Support for production level data
+- PCI/DSS and SOC2 Certified (as of 2019)
+
+### Development Sandbox
+
+The Nuxeo Dev Sandbox is a Docker based environment that enables our customers to easily build and deploy applications during their development process, like testing new configurations, addons or custom code.
```

Comment:
change this to 

The Nuxeo Development Sandbox is a Docker-container based environment that enables our customers to easily build and deploy applications during their development process. 

Nuxeo Cloud customers have the option to purchase a Development Sandbox in addition to the standard Production and Pre-Production to quickly start the application development and configuration process. Development Sandboxes include: 
- Pre-configured deployment of Nuxeo and required services
- Access to Nuxeo built CI/CD for deployments